core/converter/pipeline-converter.js

```diff
@@ -19,9 +20,8 @@ exports.PipelineConverter = Converter.specialize({
 
     _handleConvertersRangeChange: {
         value: function (plus, minus, index) {
-            var all = this.converters ? this._arrayToSet(this.converters) : [],
-                plusSet = this._arrayToSet(plus),
-                minusSet = this._arrayToSet(minus),
+            var plusSet = new Set(plus),
```

Comment:
Converting to a Set preserves the natural order of the Array when iterating on the Set?

Reply:
Confirmed with tests in collections: 
https://github.com/tejaede/collections/commit/7ff7b959c3c43fee62a2888ec67a022a9bd6b768

Reply:
If you'd like to run them as well, I found I had to manually call `npm link ../../collections` from collections/test in order to run `npm run test:jasmine` successfully.